docs/core/stories.rst

```diff
@@ -3,6 +3,8 @@
 
 .. _stories:
 
+TODO: Place clever links to rules where it makes sense?
```

Comment:
Hmm, also thinking an example of how to switch between stories & rules would be helpful (e.g. how to handle an FAQ intent/response in the middle of a story)

Reply:
what do you mean by switching? The policies will give their predictions individually and the one with the highest confidence will be chosen. If your ML policy is trained for the FAQ question, then it'll either match the RulePolicy confidence (unlikely but I think theoretically possible) or it will have a (slightly) less confident prediction and the RulePolicy wins

Reply:
I'm thinking of scenarios where the user switches to an FAQ question in the middle of a story and follow-up prompt is given to take the user back into the original story. It's something I ran into while porting the Sara bot and, from what I understand, the rule policy will have to handle the follow-up prompt but it wasn't super clear where the delineation exists between the rule & story.

Reply:
Happy to jump on a call to explain further